### PR TITLE
Feature/enable webserver

### PR DESCRIPTION
**Pull request by MCLEANS**
## Description

1. Enabled web-server to function offline.
2. Disabled web-server redirects.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Screenshots
IMG_1481

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: airrohr-firmware/airrohr-firmware.ino [hunk 1]
```diff
@@ -1985,7 +1989,7 @@ static void webserver_status() {
 	debug_outln_info(F("ws: status ..."));
 	server.sendContent(page_content);
 	page_content = F("<table cellspacing='0' border='1' cellpadding='5'>\n"
-			  "<tr><th> " INTL_PARAMETER "</th><th>" INTL_VALUE "</th></tr>");
+			"<tr><th> " INTL_PARAMETER "</th><th>" INTL_VALUE "</th></tr>");
```
Comment (chegejames) [inline airrohr-firmware/airrohr-firmware.ino]: @MCLEANS please restore this extra space.

File: airrohr-firmware/airrohr-firmware.ino [hunk 2]
```diff
@@ -1999,7 +2003,7 @@ static void webserver_status() {
 		add_table_row_from_value(page_content, F("Last OTA"), delayToString(millis() - last_update_attempt));
 	}
 #if defined(ESP8266)
-    add_table_row_from_value(page_content, F("NTP Sync"), String(sntp_time_set));
+	add_table_row_from_value(page_content, F("NTP Sync"), String(sntp_time_set));
```
Comment (chegejames) [inline airrohr-firmware/airrohr-firmware.ino]: @MCLEANS same here. Please revert to original spacing.

File: airrohr-firmware/airrohr-firmware.ino [hunk 3]
```diff
@@ -2057,6 +2061,7 @@ static void webserver_status() {
 
 	page_content += FPSTR(TABLE_TAG_CLOSE_BR);
 	end_html_page(page_content);
+
```
Comment (chegejames) [inline airrohr-firmware/airrohr-firmware.ino]: @MCLEANS please remove the extra new line.

File: airrohr-firmware/airrohr-firmware.ino [hunk 4]
```diff
@@ -1974,9 +1978,9 @@ static String delayToString(unsigned time_ms) {
  * Webserver root: show device status
  *****************************************************************/
 static void webserver_status() {
-	if (WiFi.status() != WL_CONNECTED) {
-		sendHttpRedirect();
-		return;
+	if ((WiFi.status() != WL_CONNECTED) && cfg::wifi_enabled) {	
+		sendHttpRedirect();
```
Comment (chegejames) [inline airrohr-firmware/airrohr-firmware.ino]: @MCLEANS please remove this trailing extra space.

**Review by chegejames — CHANGES_REQUESTED** [review]

**Comment by MCLEANS** [comment]
@chegejames  I have made the requested changes.

**Review by chegejames — APPROVED** [review]

**Pull request merged**